pandunia-lekse.csv

```diff
@@ -1341,6 +1362,7 @@ krize|||eng:crisis, fra:por:crise, deu:Krise, rus:кризис (krizis), spa:cri
 krizi||||critical (pertaining to crisis)||crítico (fundamental)||||||||||||||kriittinen (kriisi-)|kryzysowy
 kromium|mate|Cr||chromium||cromo|||||||||||||kromo|kromi|chrom
 krote|||rus:крот (krot)|mole (burrowing animal)||topo||||||||||||||kontiainen (maamyyrä)|kret
+krusa||||cross||cruzar|||||||||||||||
```

Comment:
Good. Need additional English translation to notify that this is a verb, like "cross (to go across)"